### ISSUE TITLE
Should the id in the geojson response now be @iot.id?

### ISSUE THREAD

**Issue by jkreft-usgs** [FraunhoferIOSB/FROST-Server#1542]
In the [Documentation for the geoJSON response](https://fraunhoferiosb.github.io/FROST-Server/extensions/GeoJSON-ResultFormat.html), the STA `@iot.id` is mapped to `id` in the `properties`.  This was the behavior in FROST 1.x.  However, in 2.x, the `@iot.id` is maintained. Is this change between 1.x and 2.x of FROST-server intentional?

Here is an example of the `@iot.id` being returned in the properties:
https://labs.waterdata.usgs.gov/sta/v1.1/Locations('000c1500-2d28-11ec-84b5-b3911c4cf066')?$resultFormat=GeoJSON

```
{
  "type": "FeatureCollection",
  "@iot.nextLink": null,
  "@iot.count": null,
  "features": [
    {
      "type": "Feature",
      "properties": {
        "@iot.id": "000c1500-2d28-11ec-84b5-b3911c4cf066",
        "name": "USGS-03026480",
        "description": "Stream",
        "encodingType": "application/vnd.geo+json",
        "properties/type": "hydrolocation",
        "properties/comid": "6873503",
        "properties/measure": 76.3882,
        "properties/comidURL": "https://geoconnex.us/nhdplusv2/comid/6873503",
        "properties/reachcode": "05010005000102",
        "properties/mainstemURL": "https://geoconnex.us/ref/mainstems/350293",
        "properties/mainstemNameAtOutlet": "Clarion River",
        "properties/downstreamMainstemURL": "https://geoconnex.us/ref/mainstems/314849",
        "properties/mainstemNameAtOutletGNIS": 1213952
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -78.5314444,
          41.6440555
        ]
      }
    }
  ]
}
```

**Comment by hylkevds** [comment]
Thanks for reporting this. Since I never got around to making tests for the GeoJSON formatter it's quite possible some more regressions slipped into the 2.x update.

**Comment by jkreft-usgs** [comment]
Good deal, thank you.  We really appreciate the geoJSON output here at USGS, we use it all the time!

**Comment by hylkevds** [comment]
GeoJSON also defines a [top-level id field](https://www.rfc-editor.org/rfc/rfc7946#section-3.2). We should be putting the id in that, for the main Entity that defines the Feature...
Though for backward compatibility I guess we should also put it in a property named id...

What do you all think?

**Comment by jkreft-usgs** [comment]
yeah, its the backwards compatibility that I am mostly concerned about.  Long term (version 3?, or as a config variable in 2.x) would probably make sense to swap toward using the top-level `id` field for the location, and then leave the `@iot.id` in the properties.   

obviously, since its an addition, the top-level id could go in at any time.

**Comment by hylkevds** [comment]
This should be fixed in the released versions 2.1.2 and 2.0.8.
